uportal-war/src/main/webapp/media/skins/respondr/common/common_skin.xml

```diff
@@ -23,6 +23,14 @@
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
+	<!-- Data Table Bootstrap Includes -->
+    <css included="plain" resource="true">/rs/datatables/1.9.4/plugins/integration/bootstrap/3/dataTables.bootstrap.css</css>
+    <css included="aggregated" resource="true">/rs/datatables/1.9.4/plugins/integration/bootstrap/3/dataTables.bootstrap.css</css>
+    
+	<!-- Data Table ColumnFilterWidgets Includes -->    <!-- Data Table ColumnFilterWidgets Includes -->
+	<css included="plain" resource="true">/rs/datatables/1.9.4/extras/column-filter-widgets/css/ColumnFilterWidgets.css</css>
+    <css included="aggregated" resource="true">/rs/datatables/1.9.4/extras/column-filter-widgets/css/ColumnFilterWidgets.css</css>
+        
```

Comment:
Do we really want to reorder these statements?  Or was this change an early attempt to remedy some missing CSS?